pages/common/xzless.md

```diff
@@ -4,6 +4,18 @@
 > See also: `less`.
 > More information: <https://manned.org/xzless>.
 
-- Open a compressed file:
+- View a compressed file:
 
 `xzless {{path/to/archive.xz}}`
+
+- View a compressed file [r]ecursively page by page:
+
+`xzless -r {{path/to/archive.xz}}`
+
```

Comment:
From `man less`:
> -r or --raw-control-chars
> Causes  "raw"  control  characters to be displayed.  The default is to display control characters using the caret notation; for example, a control-A (octal 001) is displayed as "^A" (with some exceptions as described under the -U option).
> Warning: when the -r option is  used, less cannot keep track of the actual appearance of the screen (since this depends on how the screen responds to each type of control character). Thus, various display problems may result, such as long lines being split in the wrong place.
> USE OF THE -r OPTION IS NOT RECOMMENDED.
```suggestion
```